plugins/main/common/plugin-settings.test.ts

```diff
@@ -1,6 +1,5 @@
-import { PLUGIN_SETTINGS } from './constants';
-
-describe('[settings] Input validation', () => {
+describe.skip('[settings] Input validation', () => {
+  // TODO: adapt
```

Comment:
Perhaps this test can be fixed in case we miss it in the future.

Reply:
I moved the test to the core plugin that contains the current plugin settings definition. Solved in https://github.com/wazuh/wazuh-dashboard-plugins/pull/6337/commits/5b016827e75e3f02e7b0e1fccc1aad9a3a588be1.